src/components/pages/ImageClassification/SampleUnitInputSelector/BpqObservationTypeSelector.jsx

```diff
@@ -1,54 +1,73 @@
-import React from 'react'
-import { ButtonPrimary, ButtonSecondary } from '../../../generic/buttons'
-import { H3 } from '../../../generic/text'
-import { IconSparkles, IconPen } from '../../../icons'
-import language from '../../../../language'
-import PropTypes from 'prop-types'
+import React from "react";
+import { ButtonPrimary, ButtonSecondary } from "../../../generic/buttons";
+import { H3 } from "../../../generic/text";
+import { IconPen, IconSparkles } from "../../../icons";
+import PropTypes from "prop-types";
 import {
   ButtonContainer,
   ButtonText,
   OfflineText,
   SelectorContainer,
   TextContainer,
-} from './BpqObservationTypeSelector.styles'
+} from "./BpqObservationTypeSelector.styles";
+import { Trans, useTranslation } from "react-i18next";
 
-const BpqObservationTypeSelector = ({ setIsImageClassificationSelected, isAppOnline }) => {
+const BpqObservationTypeSelector = ({
+  setIsImageClassificationSelected,
+  isAppOnline,
+}) => {
+  const { t } = useTranslation();
   const handleSampleUnitChange = (type) => () => {
-    setIsImageClassificationSelected(type)
-  }
+    setIsImageClassificationSelected(type);
+  };
 
   return (
     <SelectorContainer>
       <TextContainer>
         <H3 htmlFor="image-classification-selection">
-          {language.imageClassification.sampleUnitInputSelector.title}
+          {t("image_classification.feature_introduction.title")}
         </H3>
-        <p>{language.imageClassification.sampleUnitInputSelector.description}</p>
+        <p>
+          <Trans
+            i18nKey="image_classification.feature_introduction.description"
+            components={{
+              a: <a href="https://coralnet.ucsd.edu">CoralNet</a>,
```

Comment:
Replace the component with target here

```suggestion
a: <a href="https://coralnet.ucsd.edu" target="_blank" rel="noopener noreferrer" />
```

Reply:
Good calls!